mirrord-operator/templates/policy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- with .Values.policy -}}
+{{- if or .blockSteal .blockStealWithoutFilter .blockMirror -}}
+apiVersion: policies.mirrord.metalbear.co/v1alpha
+{{- if .isClusterWide }}
+kind: MirrordClusterPolicy
+{{- else }}
+kind: MirrordPolicy
+{{- end }}
```

Comment:
I think a namespaced policy would just be created in the current defined namespace then, right? I'm not sure how often people will actually want that.
Maybe instead of `isClusterWide` we could have a `namespace` field, that will be set in metadata when specified, and when it's not specified - a `MirrordClusterPolicy` will be created.
We could even accept an array of policies instead of just one. But those are just thoughts and we can start with this version and add the rest later if anyone wants it.

Reply:
Ah, I didn't think of that one. That's a good idea too!

I assumed anyone installing the manifest would do it with namespace included with `helm install -namespace` or under Argo CD that includes namespace, but not actually managing it themselves.

But honestly, if you think more people actually put in the namespace rather than rely on something down the road, then that's a better approach.

Reply:
I think that's a great solution @t4lz!